docs/js/Example.js

```diff
@@ -14,8 +14,8 @@ const Panel = styled.div`
 const PanelHeader = styled.div`
   height: 50px;
   padding-left: 20px;
-  background-color: ${props => props.theme.colors.athens};
-  color: ${props => props.theme.colors.gunpowder};
+  background-color: ${props => props.theme.colors.primary.purple100};
+  color: ${props => props.theme.colors.purple800};
```

Comment:
`props.theme.colors.primary.purple100` vs `props.theme.colors.purple800`?
+1 on flat! I don't like typing.

Reply:
Yeah, I was also thinking flat could be better, except that it wouldn't be obvious from the code when the theme color is being used.. Not sure about this one still... @miklosp ?

Reply:
I talked with @miklosp and we decided to flatten the main theme colors, starting from this one :)